scripts/rnnlm/lmrescore_pruned.sh

```diff
@@ -16,18 +16,16 @@ max_ngram_order=4 # Approximate the lattice-rescoring by limiting the max-ngram-
                   # the same ngram history and this prevents the lattice from 
                   # exploding exponentially. Details of the n-gram approximation
                   # method are described in section 2.3 of the paper
-                  # http://www.danielpovey.com/files/2018_icassp_lattice_pruning.pdf
-max_arcs=         # limit the max arcs in lattice while rescoring. E.g., 20000
+                  # http://www.danielpovey.com/files/2018_icassp_lattice_pruning.pdm
+max_arcs=499         # limit the max arcs in lattice while rescoring. E.g., 20000
```

Comment:
please revert these configuration changes!  Anyway you probably should have  just added the options when you invoked it.